src/node_sqlite.cc

```diff
@@ -81,6 +84,24 @@ inline MaybeLocal<Object> CreateSQLiteError(Isolate* isolate,
   return e;
 }
 
+inline MaybeLocal<Object> CreateSQLiteError(Isolate* isolate, int errcode) {
```

Comment:
The `sqlite_backup_step` function returns the errcode instead of attaching the the dest connection

https://www.sqlite.org/c3ref/backup_finish.html#sqlite3backupstep